### ISSUE TITLE
How to prepare your own training dataset?

### ISSUE THREAD

**Issue by aaronhan92** [balancap/SSD-Tensorflow#300]
Hi all,

I have few questions in mind, hoping that someone would advise me. 

1) Do i need to resize all training images to certain fixed width and height (e.g. 500 * 375) before training?
2) Assuming that, i need to resize all the training images, thereafter, I will need to manually label the ground truth value and coordinates on my resized training set in pasvoc format?
